crates/api_models/src/enums.rs

```diff
@@ -248,6 +248,121 @@ pub enum Currency {
     ZAR,
 }
 
+#[derive(
+    Clone,
+    Debug,
+    Eq,
+    Default,
+    Hash,
+    PartialEq,
+    serde::Deserialize,
+    serde::Serialize,
+    strum::Display,
+    strum::EnumString,
+    frunk::LabelledGeneric,
+    ToSchema,
+)]
+pub enum Country {
```

Comment:
You should be able to derive `Copy` as well, as long as all variants are unit variants. This would eliminate the need to clone this enum.